linux_os/guide/system/accounts/accounts-pam/password_quality/password_quality_pwquality/accounts_password_pam_retry/tests/common.sh

```diff
@@ -15,6 +15,9 @@ for file in ${configuration_files[@]}; do
 		"/etc/authselect/custom/testingProfile/$file"
 done
 authselect select --force custom/testingProfile
+{{% elif 'ubuntu' in product %}}
+rm -f /usr/share/pam-configs/*pwquality
```

Comment:
Should we remove all pwquality configs or just `cac_pwquality`?

Reply:
The container has no cac_pwquality config at default, so no need to delete those at all.